src/components/_tests_/header.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
```

Comment:
I think that this v0 can just test the rendering of the component added to page 2, and does not need to include tests for existing components that come with Gatsby